docs/src/main/asciidoc/tests-with-coverage.adoc

```diff
@@ -375,6 +400,32 @@ You can set thresholds for code coverage using the Jacoco Maven plugin. Note the
 </build>
 ----
 
+=== Gradle
+Gradle has built-in `jacoco` plugin which allows you to update the `executionData` file within the `jacocoTestCoverageVerification` task.
+
+.Example Configuration
```

Comment:
Here I would also remove the title and add the application of the jacoco plugin in the snippet

Reply:
Could you also remove these title ?